File: tests/fixtures/sample-python-project-negative/shared/utils/validators.py
import re


def validate_email(email: str) -> bool:
    email_regex = r"^[^\s@]+@[^\s@]+\.[^\s@]+$"
    return bool(re.match(email_regex, email))


def validate_name(name: str) -> bool:
    return 2 <= len(name) <= 100

File: tests/fixtures/sample-csharp-project-negative/shared/Utils/Validators.cs
using System.Text.RegularExpressions;

namespace Shared.Utils;

public static class Validators
{
    public static bool ValidateEmail(string email)
    {
        var emailRegex = new Regex(@"^[^\s@]+@[^\s@]+\.[^\s@]+$");
        return emailRegex.IsMatch(email);
    }

    public static bool ValidateName(string name)
    {
        return name.Length >= 2 && name.Length <= 100;
    }
}
